packages/flutter/test/material/menu_anchor_test.dart

```diff
@@ -3724,6 +3724,53 @@ void main() {
       },
     );
 
+    testWidgets('menus should respect overflow on right with the provided offset', (
+      WidgetTester tester,
+    ) async {
+      await changeSurfaceSize(tester, const Size(800, 600));
+      await tester.pumpWidget(
+        MaterialApp(
+          theme: ThemeData(useMaterial3: false),
```

Comment:
This can be omitted if this is not specific to M2.

Reply:
I wanted to be consistent with other tests, specifically alignment test of menu anchor.